### ISSUE TITLE
Powershell v7.2.1 Crashes with PSReadLine v2.2.0-beta1

### ISSUE THREAD

**Issue by Abisai-Arreguin** [PowerShell/PSReadLine#3143]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```
```


### Screenshot

![2022-01-09 21_34_54-Microsoft PowerShell_profile ps1 - Visual Studio Code](https://user-images.githubusercontent.com/72474489/148715712-4a91b936-2185-4c7a-9988-7ce508eb933e.png)


### Environment data

```console
PS Version: 5.1.22000.282
PS HostName: ConsoleHost (Windows Terminal)
PSReadLine Version: 2.0.0
PSReadLine EditMode: Windows
OS: 10.0.22000.1 (WinBuild.160101.0800)
BufferWidth: 120
BufferHeight: 31
```

### Steps to reproduce

Open powershell in Windows terminal.

### Expected behavior

PowerShell opens in Windows terminal.

### Actual behavior

Powershell crashes.

**Comment by Zir01** [comment]
I used VS Code recommendation to update Powershell, and I get the same error as @Abisai-Arreguin 

the only difference is mine is using PSReadLine 2.2.0-beta2 instead of beta1


### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.1
OS: Microsoft Windows 10.0.19043
BufferWidth: 120
BufferHeight: 9001

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by fuzi1996** [comment]
PowerShell: `7.2.1` and PSReadLine: `2.2.0-beta2` too!

### Environment
PSReadLine: 2.2.0-beta2
PowerShell: 7.2.1
OS: Microsoft Windows 10.0.19042
BufferWidth: 132
BufferHeight: 33

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by merddyin** [comment]
As a piece of potentially relevant additional information. I have both a Win10 and a Win11 system, each running the same version of PowerShell (7.2.1) and PSReadLine (2.1.0 Beta 2), and only the Win10 system is experiencing the error.

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta5 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta5). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` on Windows or `bash` on Linux, run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
